### ISSUE TITLE
how to backup multiple mounts on one container

### ISSUE THREAD

**Issue by ghost** [loomchild/volume-backup#16]
Hello,

  How would you suggest backup of multiple mounts on one container?  I have one container with 8 mounts to host.

Thanks.

**Comment by loomchild** [comment]
Hi. I see no particular problem - back-up each volume / mount separately. This should work regardless if volumes have names or are just mounted to host directories.

**Comment by jalbersdorfer** [comment]
# I did it like this:

## TL;DR
### Backup

```bash
docker inspect -f   '{{range .Mounts}}docker run -v {{.Name}}:/volume --rm loomchild/volume-backup backup - > {{with split .Destination "/"}} {{join . "_"}}.tar.bz2 {{println}}{{end}}{{end}}'   [containerid|name] | bash
```

### Restore
```bash
docker inspect -f '{{range .Mounts}}cat {{with split .Destination "/"}}{{join . "_"}}.tar.bz2{{end}} | docker run -i -v {{.Name}}:/volume --rm loomchild/volume-backup restore - {{println}}{{end}}' [containerid|name] | bash
```

## Explaination
I use the `docker inspect` format option to **build** fully functional backup and restore **commands** and **pipe** them into `bash` - *that's it* ;-)

@loomchild : maybe you want to add it to your README.md

**Comment by loomchild** [comment]
Cool, thanks for sharing!

I will test it and do it later.

**Comment by modem7** [comment]
> 
> # I did it like this:
> ## TL;DR
> ### Backup
> 
> ```shell
> docker inspect -f   '{{range .Mounts}}docker run -v {{.Name}}:/volume --rm loomchild/volume-backup backup - > {{with split .Destination "/"}} {{join . "_"}}.tar.bz2 {{println}}{{end}}{{end}}'   [containerid|name] | bash
> ```
> 
> ### Restore
> 
> ```shell
> docker inspect -f '{{range .Mounts}}cat {{with split .Destination "/"}}{{join . "_"}}.tar.bz2{{end}} | docker run -i -v {{.Name}}:/volume --rm loomchild/volume-backup restore - {{println}}{{end}}' [containerid|name] | bash
> ```
> 
> ## Explaination
> 
> I use the `docker inspect` format option to **build** fully functional backup and restore **commands** and **pipe** them into `bash` - _that's it_ ;-)
> 
> @loomchild : maybe you want to add it to your README.md

Are you able to expand on this a little for the noobs (aka me)?

Right now I'm doing a bash script with all individual volumes, going into individual files (bit of a bugger to update each time). 

Thanks!
